enterprise/api/v1/views.py

```diff
@@ -188,3 +195,80 @@ class EnterpriseCustomerEntitlementViewSet(EnterpriseReadOnlyModelViewSet):
     )
     filter_fields = FIELDS
     ordering_fields = FIELDS
+
+
+class EnterpriseCatalogViewSet(viewsets.ViewSet):
+    """
+    API views for `enterprise customer catalogs` api endpoint.
+    """
+    serializer_class = serializers.EnterpriseCourseCatalogReadOnlySerializer
+    permission_classes = (permissions.IsAuthenticated,)
+    authentication_classes = (OAuth2Authentication, SessionAuthentication, BearerAuthentication, JwtAuthentication)
+    throttle_classes = (ServiceUserThrottle,)
+
+    def list(self, request):
```

Comment:
Do we need this method? What's the use case for an Enterprise consumer being able to list all course catalogs?